test/unit/registerable_edition_test.rb

```diff
@@ -69,6 +69,19 @@ class RegisterableEditionTest < ActiveSupport::TestCase
     assert_equal "government/publications/edition-title", registerable_edition.slug
   end
 
+  test "sets the correct slug and routes for a deleted detailed guide" do
+    edition = create(:draft_detailed_guide, title: 'Just A Test')
+
+    Whitehall.edition_services.deleter(edition).perform!
+    assert_equal 'deleted-just-a-test', edition.slug
+
+    registerable_edition = RegisterableEdition.new(edition)
+
+    assert_equal "just-a-test", registerable_edition.slug
+    assert_equal ["/just-a-test"], registerable_edition.paths
+    assert_equal [], registerable_edition.prefixes
```

Comment:
is it worth asserting that the `registerable_edition` is archived, or is that covered elsewhere?